web/server/routers/account.ts

```diff
@@ -184,4 +186,30 @@ export const accountRouter = router({
           });
       }
     }),
+
+  getAll: protectedProcedure
+    .output(
+      z.array(
+        z.object({
+          email: z.string().email(),
+          role: z.nativeEnum(Role),
+        })
+      )
+    )
```

Comment:
Let Typescript infer the return type by removing the output, refer to subsequent comments to make this work